pkgs/dartpad_ui/lib/console.dart

```diff
@@ -118,6 +119,61 @@ class _ConsoleWidgetState extends State<ConsoleWidget> {
                   ],
                 ),
               ),
+              // If there is a JS error, and the app type is Flutter, show a
+              // "File an issue" button. JS errors are normal when executing
+              // Dart (non-Flutter) code.
+              if (widget.output.hasError &&
+                  widget.output.hasJavascriptError &&
+                  appModel.appIsFlutter.value == true)
+                Align(
+                  alignment: Alignment.bottomRight,
+                  child: Row(
+                    mainAxisAlignment: MainAxisAlignment.end,
+                    children: [
+                      Text(
+                        'Oops! Something went wrong.',
+                        style: TextStyle(
+                          color: Theme.of(context).colorScheme.tertiary,
+                        ),
+                      ),
+                      SizedBox(width: 12),
+                      TextButton(
+                        style: ButtonStyle(
+                          backgroundColor: WidgetStateProperty.all(switch (theme
+                              .brightness) {
+                            Brightness.light =>
+                              theme.colorScheme.surface.darker,
+                            _ => theme.colorScheme.surface.lighter,
+                          }),
+                          foregroundColor: WidgetStateProperty.all(
+                            theme.colorScheme.onPrimary,
+                          ),
+                        ),
+                        child: Text('File an issue'),
+                        onPressed: () {
+                          final issueBody = _createGithubIssue(
+                            widget.output.error,
+                          );
+                          const labels = 'type-bug';
```

Comment:
I think non-committers to a repo can't actually assign labels when creating issues (they could is using an issue template). Instead of a label, perhaps we have a common `[console error] unexpected javascript error from running app` prefix for the title?